apps/documents/rules.py

```diff
@@ -0,0 +1,16 @@
+import rules
+from rules.predicates import is_superuser
+
+from adhocracy4.modules.predicates import is_context_initiator
+from adhocracy4.modules.predicates import is_context_member
+from adhocracy4.modules.predicates import is_context_moderator
+from adhocracy4.modules.predicates import is_public_context
+
+
+rules.add_perm('meinberlin_documents.view',
+               is_superuser | is_context_moderator | is_context_initiator |
+               is_context_member | is_public_context)
+
+
+rules.add_perm('meinberlin_documents.create',
+               is_superuser | is_context_moderator | is_context_initiator)
```

Comment:
> Why does a moderator have this permission?

Can think about it. Makes sense, somehow, as there could be one initiator that created the project, but multiple user contributing to the document.